wiki/Article_Styling_Criteria/en.md

```diff
@@ -437,15 +382,14 @@ The osu! official branding are to not use any spaces. Examples may include:
 - `osu!taiko`
 - `osu!mania`
 - `osu!direct`
-- `osu!wiki`
 
 Everything else are to use spaces. For example:

```

Comment:
L386: consider Revising for clarity ``[..] Other brandings that isn't covered by the official branding guidelines should use spaces, for example: [...]``